docs/source/development/adding_to_the_api_documentation.rst

```diff
@@ -0,0 +1,324 @@
+Adding to the API Documentation
+===============================
+
+Documentation is an integral part of every collaborative software
+project. Good documentation not only encourages users of the package to
+try out different functionalities, but it also makes maintaining and
+expanding code significantly easier. Every code contribution to the
+package must come with appropriate documentation of the API. This guide
+details how to do this.
+
+Docstrings
+----------
+
+The main form of documentation are docstrings, multi-line comments
+beneath a class or function definition with a specific syntax, which
+detail its functionality. This package uses the `NumPy docstring
+format <https://numpydoc.readthedocs.io/en/latest/format.html#numpydoc-docstring-guide%3E>`__.
+As a rule, all functions which are exposed to the user *must* have
+appropriate docstrings. Below is an example of a docstring for a
+probabilistic numerical method.
+
+.. code:: ipython3
+
+    # %load -r 1-163 ../../../src/probnum/linalg/_problinsolve.py
```

Comment:
Here instead of `code` the directive `literalinclude` should allow the same functionality as the `%load` statement which imports the lines `1-163` from the corresponding file.

Reply:
During Jupyter notebook to RST conversion, `literalinclude` replaces `%load` command in the commit https://github.com/probabilistic-numerics/probnum/pull/676/commits/42d0f74d5cc3826c3f97e96f37d3512c8508e703.